### DISCUSSION TITLE
hydrogen atoms with zero radius ?!

### DISCUSSION THREAD

**Discussion by UnixJunkie** [Electrostatics/pdb2pqr#347] [Q&A]
Hello,

While processing PDB:3RFM, I noticed that in the output .pqr file, some Hs have a radius of 0.0000.
Why?
Should I be worried (I already am, honestly)?

I used https://server.poissonboltzmann.org/pdb2pqr
but also pdb2pqr (Version 2.1.1) from the command line.

Thanks a lot,
Francois.

**Comment by sobolevnrm** [comment]
Some force fields (e.g., PARSE) use zero-radius parameters for some atoms, particularly hydrogen in some environments.  This generally occurs when the radius of the heavy atom is large enough to encompass the entirety of the hydrogen atom, eliminating the need for a non-zero hydrogen radius.

**Reply by UnixJunkie** [reply]
Interesting; I did not know that. I think I used the amber FF.
